src/reporting/market.se

```diff
@@ -44,10 +45,13 @@ data tentativeWinningPayoutDistributionHash
 data finalPayoutDistributionHash
 data automatedReporterPayoutDistributionHash
 data automatedReporterDisputeBondHolder
+data automatedReporterDisputeBondToken
```

Comment:
No, I'm saying keep `automatedReporterDisputeBondToken` and get rid of `automatedReporterPayoutDistributionHash` and `automatedReporterDisputeBondHolder`.  Repeat for the other bond tokens.  That data is now stored inside the bond token, so we only need the bond token address to have access to that data.